### ISSUE TITLE
Rendering error in Obsidian document when excalidraw's alias contains spaces

### ISSUE THREAD

**Issue by 1-2-3** [zsviczian/obsidian-excalidraw-plugin#387]
**Describe the bug**
Rendering error in Obsidian document when excalidraw's alias contains spaces。
Error message displayed in Developer Tools:
```log
app.js:1 Uncaught (in promise) DOMException: Failed to execute 'add' on 'DOMTokenList': The token provided ('excalidraw-svg-test aliases1') contains HTML space characters, which are not valid in tokens.
    at HTMLImageElement.Element.addClasses (app://obsidian.md/app.js:1:131198)
    at HTMLImageElement.Element.addClass (app://obsidian.md/app.js:1:131100)
    at getIMG (eval at <anonymous> (app://obsidian.md/app.js:1:1357475), <anonymous>:180:8628)
    at createImageDiv (eval at <anonymous> (app://obsidian.md/app.js:1:1357475), <anonymous>:180:9690)
    at createImgElement (eval at <anonymous> (app://obsidian.md/app.js:1:1357475), <anonymous>:180:13035)
    at eval (eval at <anonymous> (app://obsidian.md/app.js:1:1357475), <anonymous>:180:13355)
```

**To Reproduce**
1. Create an Excalidraw drawing, open it as a Markdown file, and add an alias with spaces:
```md
---
excalidraw-plugin: parsed
aliases: test aliases1
tags: 
---
==⚠  Switch to EXCALIDRAW VIEW in the MORE OPTIONS menu of this document. ⚠==

# Text Elements
```
2. Create a Markdown document in Obsidian, referencing the Excalidraw drawing created in the first step:
```md
![[Drawing 2022-0115-2217.excalidraw|test aliases1]]
```
Then enter preview mode, the drawing cannot be rendered.
3. Remove the space in the alias in the link, enter preview mode and the drawing can be rendered:
```
![[Drawing 2022-0115-2217.excalidraw|testaliases1]]
```

**Expected behavior**
Excalidraw drawing aliases can also be rendered with spaces

**Desktop (please complete the following information):**
 - Obsidian: v0.13.19
 - Excalidraw Plugin: v1.5.21

**Comment by zsviczian** [comment]
That is because the alias is treated as a style and added to the <img> tag in the rendered HTML page. I will escape the the style to avoid this error.

**Comment by zsviczian** [comment]
resolved in 1.5.22
